SpywareFilter/sections/specific.txt

```diff
@@ -107,6 +107,7 @@ semprotyuk.com##div[style^="margin-bottom:150px;"][style*="padding-right:2em;"]
 ||hankyung.com/resource/js/w/view.gtmet.js
 ||cdn.thunderhead.com/one/rt/js/one-tag.js
 ||discord.com/api/v*/science
+/flying-analytics/js/analytics.js
```

Comment:
please add $domain

Reply:
It looks popular
https://publicwww.com/websites/%22%2Fflying-analytics%2Fjs%2Fanalytics.js%22/

Reply:
@Alex-302 so to general_url.txt ?

Reply:
@wout-codes @Sergey-Lyapin  Yes